input/docs/fundamentals/dependencies.md

```diff
@@ -19,3 +20,25 @@ RunTarget("B");
 ```
 
 This will first execute target `A` and then `B` as expected.
+
+
+# Reverse task dependencies
+
+Since version 0.23.0, if you prefer define dependencies with a reversed relationship, you can define it using the `IsDependeeOf` method.
```

Comment:
@gsferreira just a minor correction...

Think this would read better as `if you prefer to define` and also `you can define them using the`